src/create_shards.c

```diff
@@ -378,6 +380,8 @@ ParseWorkerNodeFile(char *workerNodeFilename)
 	char workerNodeLine[MAXPGPATH];
 	char *workerFilePath = make_absolute_path(workerNodeFilename);
 	char workerLinePattern[1024];
+	const int WorkerNameIndex = 0;
+	const int WorkerPortIndex = 1;
```

Comment:
We generally use lowercase letters as the initial letter.  But, I think this uppercase letters come from CitusDB and we should stick to that?